opencensus_shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryCtx.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opencensusshim;
+
+import io.opencensus.trace.ContextHandle;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+
+public class OpenTelemetryCtx implements ContextHandle {
+
+  private final Context context;
+  private Scope scope;
+
+  public OpenTelemetryCtx(Context context) {
+    this.context = context;
+  }
+
+  Context getContext() {
+    return context;
+  }
+
+  @Override
+  public ContextHandle attach() {
+    scope = context.makeCurrent();
+    return this;
+  }
+
+  @Override
+  public void detach(ContextHandle ctx) {
+    OpenTelemetryCtx impl = (OpenTelemetryCtx) ctx;
+    impl.scope.close();
```

Comment:
Null check for impl.scope - if the user did incorrectly use scope and introduce a threading problem, it could be null here.